### DISCUSSION TITLE
Error: Action "click->hw-combobox#selectOnClick" references undefined method "selectOnClick"

### DISCUSSION THREAD

**Discussion by timmyd87** [josefarias/hotwire_combobox#121] [Q&A]
Getting an error in console when clicking on an option in the list - works fine if selecting via 'tab'. The option does not select when clicked, only on tab.

my combobox looks like:

` <%= form.combobox :location_id, Location.all, placeholder: "Select location" %>`

running

GIT
  remote: https://github.com/josefarias/hotwire_combobox.git
  revision: 002eb00c851fc179f51e5f1f7125101c4040562f
  branch: main
  specs:
    hotwire_combobox (0.2.0)
      rails (>= 7.0.7.2)
      stimulus-rails (>= 1.2)
      turbo-rails (>= 1.2)

but producing
![Screenshot 2024-03-26 at 11 57 07 AM](https://github.com/josefarias/hotwire_combobox/assets/12996091/7a909de2-1270-4f41-b6b8-ef82a25ae5e1)

Same behaviour for an async box too. Not sure if this is me but it can see that method defined in selection.js

**Comment by josefarias** [comment]
Hey @timmyd87, are you also using the npm package in addition to the gem? Is it possible you updated the gem but not the npm package? They should both be running the same version every time.

Let me know. Thanks!

**Reply by timmyd87** [reply]
Yeah i learnt my lesson from last time but still an issue

running version 0.1.43 for the JS

**Reply by josefarias** [reply]
Ah right, so you should be running 0.2.0 on JS as well as the gem. 0.2.0 was released on npm 4 days ago so I think that's what's going on https://www.npmjs.com/package/@josefarias/hotwire_combobox

Can you update to v0.2.0 of the npm package please and let me know if that fixes it for you? Thanks!

**Reply by timmyd87** [reply]
Yeah running yarn upgrade @josefarias/hotwire_combobox didn't work but manually changed to 0.2.0 and we are good

Sorry @josefarias but thank you for the help

**Reply by josefarias** [reply]
No worries! Glad that did it. Closing for now 👍
